hawkbit-rest/hawkbit-rest-docs/src/test/java/org/eclipse/hawkbit/rest/ddi/documentation/RootControllerDocumentationTest.java

```diff
@@ -580,4 +581,38 @@ public void getControllerInstalledBaseAction() throws Exception {
 
     }
 
+    @Test
+    @Description("Feedback channel for confirming an action")
+    @WithUser(tenantId = "TENANT_ID", authorities = "ROLE_CONTROLLER", allSpPermissions = true)
+    public void postConfirmationFeedback() throws Exception {
```

Comment:
What about the confirmationBase GET endpoint?

Reply:
Implemented another test case for that